doc/api/worker_threads.md

```diff
@@ -360,11 +360,8 @@ Notable differences inside a Worker environment are:
   being invoked.
 - IPC channels from parent processes are not accessible.
 - The [`trace_events`][] module is not supported.
-
-Currently, the following differences also exist until they are addressed:
-
-- The [`inspector`][] module is not available yet.
```

Comment:
Just as question: is that resolved in here as well (or was it already resolved)? I could not really see any references to the inspector in here.

Reply:
It already was resolved last year, and was pointed out when @vsemozhetbyt reviewed this PR.